miden/src/repl/mod.rs

```diff
@@ -245,8 +250,8 @@ fn execute(program: String) -> Result<(Vec<(u64, Word)>, Vec<Felt>), MidenError>
 
     let pub_inputs = vec![];
     let inputs = ProgramInputs::new(&pub_inputs, &[], vec![]).unwrap();
-    let mut process = Process::new_debug(inputs);
-    let _program_outputs = process
+    let mut process = Process::new_debug(program.kernel(), inputs);
+    let _program = process
```

Comment:
Why did you change this from `_program_outputs` to `_program`? I think the previous name was correct.

Reply:
Reverted it back to old name.